ush/get_extrn_mdl_file_dir_info.sh

```diff
@@ -398,6 +398,8 @@ fi
 
         if [ "${MACHINE}" = "JET" ] || [ "${MACHINE}" = "HERA" ] || [ "${MACHINE}" = "ORION" ]; then
           fns_on_disk=( "${yy}${ddd}${hh}0${fcst_mn}0${fcst_hh}" )
+        elif [ "${MACHINE}" = "HERA" ] ; then
```

Comment:
remove `[ "${MACHINE}" = "HERA" ]` from line 399

Reply:
The data path and filename of FV3GFS are different from what we have on Jet. That is why I made these changes. Has anybody tested with FV3GFS real-time data on HERA with the old setting?

Reply:
You added [ "${MACHINE}" = "HERA" ] in Line 401 but did not remove it from line 399, so Line 401 will not execute.

Reply:
Done with the change. Thank you!